react/components/HomeMetadata/components/HomeBreadcrumb/index.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import styles from 'react/styles';
+
+import StickyBreadcrumbPath from 'react/components/UI/StickyBreadcrumbPath';
+import WithCurrentRoute from 'react/hocs/WithCurrentRoute';
+
+const Options = styled.div`
+`;
+
+const Option = styled.a`
+  display: block;
+
+  &:last-child {
+    color: ${styles.Colors.gray.regular};
+
+    &:hover {
+      color: ${styles.Colors.gray.semiBold};
+    }
+  }
+`;
+
+const feedOptions = (
+  <Options>
+    <Option>Feed</Option>
+    <Option href="/explore">Explore</Option>
+  </Options>
+);
+
+const exploreOptions = (
+  <Options>
+    <Option>Explore</Option>
+    <Option href="/">Feed</Option>
+  </Options>
+);
+
+class HomeBreadcrumb extends Component {
+  static propTypes = {
+    currentRoute: PropTypes.shape({
+      href: PropTypes.string.isRequired,
+    }).isRequired,
+  }
+
+  render() {
+    const { currentRoute: { pathname } } = this.props;
+
+    const stuckChildren = (
+      <StickyBreadcrumbPath.Crumb>
+        {{
+          '/': 'Feed',
+          '/explore': 'Explore',
+          '/explore/channels': 'Explore',
+          '/explore/blocks': 'Explore',
+          '/notifications': 'Notifications',
+        }[pathname]}
+      </StickyBreadcrumbPath.Crumb>
+    );
+
+    return (
+      <StickyBreadcrumbPath stuckChildren={stuckChildren}>
+        <StickyBreadcrumbPath.Crumb>
+          {{
+            '/': feedOptions,
+            '/explore': exploreOptions,
+            '/explore/channels': exploreOptions,
+            '/explore/blocks': exploreOptions,
+            '/notifications': 'Notifications',
+          }[pathname]}
+        </StickyBreadcrumbPath.Crumb>
+      </StickyBreadcrumbPath>
+    );
+  }
+}
+
+export default WithCurrentRoute(HomeBreadcrumb);
```

Comment:
We: really need a router. And: these need to be separate apps.

Reply:
I would recommend looking at [found](https://github.com/4Catalyzer/found). Its very flexible

Reply:
Weirdly, i can't find a comparison with react-router. Have you used both?

Reply:
Yeah -- This is basically a more customizable react router 3, with much of the same API. React router 4 was a complete API redesign which made static route declarations 2nd class and thus optimizations like bundle splitting at the route level, etc, kinda hacky. For most common usecases tho it's basically RR3, just with all of its internals exposed should you want to adapt it to complex app env.